OpenOversight/app/main/views.py

```diff
@@ -794,6 +799,9 @@ def edit_department(department_id: int):
         department.state = form.state.data
         department.last_updated_by = current_user.id
         db.session.flush()
+        Department(id=department_id).remove_database_cache_entries(
+            [KEY_DEPTS_BY_STATE],
+        )
```

Comment:
Remove cache entry for departments by state when editing a department.

Reply:
There's an argument for only clearing it when you modify the state, but I don't think it's worth the extra control statements.